docs/docs/modules/agents/how_to/custom_llm_chat_agent.mdx

```diff
@@ -2,7 +2,7 @@
 
 This notebook explains how to create your own custom agent based on a chat model.
 
-An LLM chat agent consists of four key components:
+An LLM chat agent consists of three key components:
```

Comment:
@fvaysh Sorry, I was not clear. There are 4 key components in both cases, not 3.